dapps/js-glue/src/codegen.rs

```diff
@@ -102,7 +102,7 @@ fn implement_webapp(cx: &ExtCtxt, builder: &aster::AstBuilder, item: &Item, push
 	let files_impl = quote_item!(cx,
 		impl $type_name {
 			fn files() -> ::std::collections::HashMap<&'static str, File> {
-				let mut files = ::std::collections::HashMap::new();
```

Comment:
That won't work if `$statements` are not empty. Actually it shouldn't be printed while compiling Parity at all (since we always have some files there).